ci/compute-matrix.jq

```diff
@@ -1,14 +1,21 @@
 def compute_arch($x):
-  ["amd64"] |
-  if
-    $x.CUDA_VER > "11.2.2" and
-    $x.LINUX_VER != "centos7"
-  then
-    . + ["arm64"]
+  if $x.IMAGE_REPO != "ci" then
+    ["amd64"] |
+    if ["ubuntu18.04", "centos7"] | index($x.LINUX_VER) != null then
+      .
+    else
+      . + ["arm64"]
+    end |
+    $x + {ARCHES: .}
   else
-    .
-  end |
-  $x + {ARCHES: .};
+    ["amd64"] |
+    if $x.CUDA_VER > "11.2.2" and $x.LINUX_VER != "centos7" then
+      . + ["arm64"]
+    else
+      .
+    end |
+    $x + {ARCHES: .}
+  end;
```

Comment:
I think we can remove the top-level `if`/`else` block here to simplify this conditional statement.

Combinations consisting of the following components only have `amd64` variants:

- `LINUX_VER` == `ubuntu18.04`
- `LINUX_VER` == `centos7`
- `CUDA_VER` <= `11.2.2`